### DISCUSSION TITLE
Is there a way to completely remove OAuth apps from organization?

### DISCUSSION THREAD

**Discussion by piotrekkr** [community/community#123319] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello. Is there a way to remove completely OAuth app that was added to GithHub organization? Disabled some OAuth apps years ago and don't want to see those on the list. 

I cannot find any option to remove such apps. Is it possible and if not is there any specific reason?

Thanks

![image](https://github.com/community/community/assets/503782/98491f00-5c04-4764-8c35-53a612ad66b1)

**Comment by davevad93** [comment, marked as answer]
Hi @piotrekkr ,

As far as I know, it isn't possible to completely remove an OAuth app from your organization, denying access to it is the only option. It completely removes access to the apps from your organization but unfortunately those apps will still appear in your organization's list as denied. Honestly, I don't know why.

**Comment by ebndev** [comment]
Hey @piotrekkr - can you see if the steps in this doc helps? [Deleting an OAuth app
](https://docs.github.com/en/apps/oauth-apps/maintaining-oauth-apps/deleting-an-oauth-app)

**Reply by piotrekkr** [reply]
Hello. Thanks for info but I was asking about third party OAuth apps (check my screenshot), not apps created by me. I know that I can can remove those myself.

**Reply by ebndev** [reply]
Oh, oops sorry I missed that part!
